src/Native/pkg/runtime.native.System.Net.Http/runtime.native.System.Net.Http.builds

```diff
@@ -27,6 +27,10 @@
       <OSGroup>osx.10.10</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Include="opensuse\13.2\runtime.native.System.Net.Http.pkgproj">
+      <OSGroup>opensuse.13.2/OSGroup>
```

Comment:
Thanks, Steve.  Not really sure why this was working locally in my internals builds.  I've fixed up the issue on TFS.  I'm going to close this so the mirror picks up the new commit.